ftw/contacts/profiles/geo/types/ftw.contacts.Contact.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<object name="ftw.contacts.Contact" meta_type="Dexterity FTI"
+        i18n:domain="ftw.contacts" xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+
+  <property name="behaviors" purge="False">
+      <element value="collective.geo.behaviour.interfaces.ICoordinates" />
+  </property>
```

Comment:
It seems not...

The ICoordinates is a behaviour
The IGeoreferenceable is only a markerinterface

The ICoordinates-behaviour marks the content with the IGeoreferenceable interface
```
    <plone:behavior
        title="Collective Geo Maps"
        description="This behaviour will make a content geo referenceable"
        provides=".interfaces.ICoordinates"
        marker="collective.geo.geographer.interfaces.IGeoreferenceable"
        factory=".behaviour.Coordinates"
        />
```